packages/wonder-stuff-ci/src/extract-mobile-release-info-from-branch-name.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Extract release version and prefix from mobile release branch.
+ *
+ * Example, given the branch `release/unified/v7.8.0`:
+ * {
+ *    prefix: "release/unified/",
+ *    version: "7.8.0"
+ * }
+ *
+ * @param {string} The release branch of the form `[release/[unified|ios|android]]/[v]<num>.<num>.<num>[-extra]`.
+ * @returns {Object} The release version and prefix, if found; otherwise, `null`.
```

Comment:
**suggestion:**  If you create the type as suggested below and then replace `{Object}` with `{MobileReleaseInfo}`, which gives folks a little nicer info in their IDEs.